### PR TITLE
Core 1.6 change log

### PR DESCRIPTION
**Pull request by Goslinl**

### CODE REVIEW

File: guides/CopyofFHIRImplementationGuideTemplateTest/Home/Core/1.0.3/Security-and-Authorisation/Receiver.page.md
Comment (DylanHainsworthNHSD) [inline guides/CopyofFHIRImplementationGuideTemplateTest/Home/Core/1.0.3/Security-and-Authorisation/Receiver.page.md:1]: Not sure about the use of the word "Current" as of 04/06 those will be obsolete. Can we call them "Original" or "Old" and call the above 2 "Latest" or "New" ?

**Review by DylanHainsworthNHSD — CHANGES_REQUESTED** [review]
Not sure about the use of the word "Current" as of 04/06 those will be obsolete. Can we call them "Original" or "Old" and call the above 2 "Latest" or "New" ?

File: guides/CopyofFHIRImplementationGuideTemplateTest/Home/About-BaRS/Releases/Technical-Release-Notes/BaRS-Core/1.1.3.page.md
Comment (julieharris37) [inline guides/CopyofFHIRImplementationGuideTemplateTest/Home/About-BaRS/Releases/Technical-Release-Notes/BaRS-Core/1.1.3.page.md:1]: Documentation for BaRS Foundations - at the Hazard workshop DR said we should call it Standard Pattern for Appointment Management as it isn't BaRS specific. Is this still the right terminology?

Comment (Goslinl) [reply guides/CopyofFHIRImplementationGuideTemplateTest/Home/About-BaRS/Releases/Technical-Release-Notes/BaRS-Core/1.1.3.page.md:1]: i can rename it to this, no problem

File: guides/CopyofFHIRImplementationGuideTemplateTest/Home/Core/1.0.3/Security-and-Authorisation/Receiver.page.md
Comment (Goslinl) [reply guides/CopyofFHIRImplementationGuideTemplateTest/Home/Core/1.0.3/Security-and-Authorisation/Receiver.page.md:1]: Old would indicate they are still okay, expired/deprecated explicitly defines their state. the top 2 are the old, the bottom two are the new.

File: guides/CopyofFHIRImplementationGuideTemplateTest/Home/About-BaRS/Releases/Technical-Release-Notes/BaRS-Core/1.1.3.page.md
Comment (Goslinl) [reply guides/CopyofFHIRImplementationGuideTemplateTest/Home/About-BaRS/Releases/Technical-Release-Notes/BaRS-Core/1.1.3.page.md:1]: have updated all references to Foundation to Standard Pattern including documentation itself and core release notes.

**Review by DylanHainsworthNHSD — APPROVED** [review]

File: guides/CopyofFHIRImplementationGuideTemplateTest/Home/Core/1.0.3/Security-and-Authorisation/Receiver.page.md
Comment (Goslinl) [reply guides/CopyofFHIRImplementationGuideTemplateTest/Home/Core/1.0.3/Security-and-Authorisation/Receiver.page.md:1]: wasnt in 1.1.3 though! have added.

**Pull request merged**